astropy/io/ascii/ipac.py

```diff
@@ -214,6 +216,10 @@ def get_cols(self, lines):
                 null = header_vals[3][i].strip()
                 fillval = "" if issubclass(col.type, core.StrType) else "0"
                 self.data.fill_values.append((null, fillval, col.name))
+            if "long".startswith(col.raw_type.lower()):
```

Comment:
I'm slightly confused by this line. Is it not doing more than it should ? Is, e.g. `"Lo"` a correct way to specify `long` ints ?

Reply:
I'm not 100% sure `long` can be abbreviated as `Lo` per [IPAC spec](https://irsa.ipac.caltech.edu/applications/DDGEN/Doc/ipac_tbl.html#datatypes), but current astropy implementation does allow all kind of abbreviation.

The above line mimics the existing ones in `get_col_type()` in https://github.com/astropy/astropy/blob/d1579ecbce54be7aa22dcf7cfb17b6cb1c537fbe/astropy/io/ascii/ipac.py#L165 :

The data type abbreviation has been explicitly tested in:https://github.com/astropy/astropy/blob/d1579ecbce54be7aa22dcf7cfb17b6cb1c537fbe/astropy/io/ascii/tests/test_read.py#L1367-L1370

Reply:
great, thank you very much for clarifying this !

Reply:
Unresolving just so that others can see this too!